### PR TITLE
Cp 341 fix dev permissions

### PR DESCRIPTION
**Pull request by Hatscat**
## 📝 What is this PR

-   [Specs](https://phantombuster.atlassian.net/browse/341)

### CODE REVIEW

File: manifest.dev.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
 	"name": "PhantomBuster DEV",
-	"optional_permissions": ["*://*.phantomcompany.com/*", "*://*.phantomcompany.io/*", "*://localhost/*"],
+	"optional_host_permissions": ["*://*.phantomcompany.com/*", "*://*.phantomcompany.io/*", "*://localhost/*"],
```
Comment (RiahiKarim) [inline manifest.dev.json:3]: Isn't `optional_host_permissions` irrelevant without `optional_permissions`?

My understanding is that `optional_host_permissions` is like a complementary attribute for `optional_permissions`. In a similar manner `host_permissions` is an attributes that goes with `permissions`. 

https://developer.chrome.com/docs/extensions/mv3/mv3-migration/#host-permissions

Comment (Hatscat) [reply manifest.dev.json]: I commit this last suggestion, let me know what you think :)

**Review by RiahiKarim — APPROVED** [review]
LGTM 💯

**Review by LoganPB — APPROVED** [review]
LGTM 💯

**Pull request merged**